docs/CHANGES_4.0.0.md

```diff
@@ -1,3 +1,39 @@
 # Changes in 4.x
 
 WIP
+
+## Versioned API
+
+Versioned API is a new feature in MongoDB 5.0 that allows user-selectable API versions, subsets of MongoDB server semantics, to be declared on a client. During communication with a server, clients with a declared API version will force the server to behave in a manner compatible with the API version. Declaring an API version on a client can be used to ensure consistent responses from a server, providing long term API stability for an application. The declared API version is applied to all commands run through the client, including those sent through the generic RunCommand helper. Specifying versioned API options in the command document AND declaring an API version on the client is not supported and will lead to undefined behaviour.
```

Comment:
Not sure if this is the right place, but maybe it’s worth mentioning that setting the option on a pre-5.0 server fails?